internal/pkg/dto/coreValues.go

```diff
@@ -0,0 +1,40 @@
+package dto
+
+type UpdateQueryRequest struct {
```

Comment:
Should create separate Repo and Service structs. We don't return all the data that we store in DB. Some fields are for internal use only.

Reply:
Done